db/migrate/20230623120905_migrate_organization_taxes.rb

```diff
@@ -0,0 +1,206 @@
+# frozen_string_literal: true
+
+class MigrateOrganizationTaxes < ActiveRecord::Migration[7.0]
+  # NOTE: redifine models to prevent schema issue in the future
+  class Organization < ApplicationRecord; end
+  class Tax < ApplicationRecord; end
+
+  class Customer < ApplicationRecord; end
+  class CustomersTax < ApplicationRecord; end
+
+  class FeesTax < ApplicationRecord; end
+  class InvoicesTax < ApplicationRecord; end
+  class CreditNotesTax < ApplicationRecord; end
+
+  def change
+    reversible do |dir|
+      dir.up do
+        # NOTE: migrate organizations taxes
+        Organization.where('vat_rate > 0').find_each do |organization|
+          Tax.create_with(
+            organization_id: organization.id,
+            name: "Tax (#{organization.vat_rate}%)",
+            rate: organization.vat_rate,
+            applied_to_organization: true,
+          ).find_or_create_by!(code: "tax_#{organization.vat_rate}")
+        end
+
+        # NOTE: migrate customers taxes
+        Customer.where.not(vat_rate: nil).find_each do |customer|
+          tax = Taxe.create_with(
+            name: "Tax (#{customer.vat_rate}%)",
+            rate: customer.vat_rate,
+          ).find_or_create_by!(
+            organization_id: customer.organization_id,
+            code: "tax_#{customer.vat_rate}",
+          )
+
+          CustomersTax.create!(
```

Comment:
```suggestion
          Customer::AppliedTax.create!(
```

Reply:
The class is defined here: https://github.com/getlago/lago-api/pull/1130/files#diff-ff78ad818a92299500a4d5cc5b5b7f66a20c201e39eb394c474f0202776af01cR9

I didn't wanted to use the model class, to prevent the errors we add lately with schema changes, and I choose to follow the rails default for the naming to make it simpler (and limit confusion :thinking: ).

If you prefer I could use the same naming as the model